aptos-move/block-executor/src/executor.rs

```diff
@@ -430,19 +425,12 @@ where
                     executor,
                     base_view,
                     ParallelState::new(versioned_cache, scheduler, shared_counter),
-                );
-                if record_succeeded {
-                    scheduler.finish_execution_during_commit(txn_idx);
-                } else {
-                    // TODO add support
-                    panic!("module re-validation in commit retry");
-                    // if scheduler.halt() {
-                    //     *maybe_error = Some(Error::ModulePathReadWrite);
-                    // }
-                    // SchedulerTask::NoTask
-                }
+                )?;
```

Comment:
Here, we are not checking the execution status returned by Self::execute. Should we check it explicitly for the sake of completeness?

Reply:
we do check result (via ? )

I can add the comment, but we don't need to check update_outside, as we move validation index irrespectively